docs/docs/workflows/sharing/README.md

```diff
@@ -0,0 +1,109 @@
+</div>
+
+
+The copied workflow will include the same trigger, steps and connected account configuration, but it will have a separate event history and versioning from the original.
+
+## Creating a share link for a workflow
+
+To share a workflow, open the **Builder** for the workflow. Then in the top right menu, select **Create Share Link**.
+
+![Click "Create Share Link" in the workflow's settings within the builder to generate a sharable link](https://res.cloudinary.com/pipedreamin/image/upload/v1685119418/docs/docs/share%20workflows/CleanShot_2023-05-26_at_12.42.22_p4q3dr.png)
+
+Now you can define which prop values should be included in this shareable link.
+
+## Including props
+
+Optionally, you can include the actual individual prop configurations as well. This helps speed up workflow development if the workflow relies on specific prop values to function properly.
+
+You can choose to **Include all** prop values if you'd like, or only select specific props.
+
+For the daily schedule reminder workflow, we included the props for filtering Google Calendar events, but we did _not_ include the SMS number to send the message to. This is because the end user of this workflow will use their own phone number instead:
+
+![Sharing a workflow that will send a daily SMS message of your Google Calendar schedule for today](https://res.cloudinary.com/pipedreamin/image/upload/v1685113542/docs/docs/share%20workflows/CleanShot_2023-05-26_at_11.05.16_hebqpl.png)
+
+::: tip Connected Accounts are not shared
+
+Shared workflow links **will not** include your own connected accounts. Instead in this new workflow the user of your workflow link will be prompted to connect their own accounts.
+
+:::
+
+## Versioning
+
+When you create a shared link for your workflow, that link is frozen to the version of your workflow at the time the link was created.
+
+If changes are made to the original workflow, the changes will _not_ be included in the shared workflow link, nor in any workflows copied from the original shared link.
+
+[Generate a new share link](#creating-a-share-link-for-a-workflow) to include new changes to a workflow.
+
+::: tip Share Links persist
+
+You can create multiple share links for the same workflow with different prop configurations, or even different steps.
+
+Share links will not expire or be overridden.
+
+:::
+
+## Frequently Asked Questions
+
+### If changes are made to the original workflow, will copied versions of the workflow from the shared link also change?
+
+No, workflows copied from a shared link will have separate version histories from the original workflow. You can modify your original workflow and it will not affect copied workflows.
+
+### Will my connected accounts be shared with the workflow?
+
+No, your connected accounts **will not** be shared. Instead copied workflows will display a slot in actions that required a connected account, so the user of the copied workflow can provide their own accounts instead.
```

Comment:
```suggestion
No, your connected accounts are not shared. Instead, copied workflows display a slot in actions that require a connected account, so the user of the copied workflow can provide their own accounts instead.
```

Reply:
Changed "will not" to "are not" to follow Google's style guide on present tense.